README.md

```diff
@@ -152,6 +152,33 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 }
 ```
 
+### RationaleCallbacks
+Implement this callback if you want to interact with the rationale dialog.
+
+```java
+public class RationaleFragment extends Fragment implements EasyPermissions.RationaleCallbacks {
+
+    @Override
+    public void onRationaleAccepted(int requestCode) {
+        // Rationale accpets to request some permissions
+        // ...
+    }
+
+    @Override
+    public void onRationaleDenied(int requestCode) {
+        // Rationale denied to request some permissions
+        // ...
+    }
+}
+```
+
+Check out [RationaleFragment](https://github.com/googlesamples/blob/master/app/src/main/java/pub/devrel/easypermissions/sample/RationaleFragment.java) in the sample app folder for the full example.
```

Comment:
The link can be simplified to `app/src/main/java/pub/devrel/easypermissions/sample/RationaleFragment.java`. GitHub knows the project's directory structure. 👍

Also, it could be reworded to "For a full example, take a look at the sample RationaleFragment."

Reply:
👍